app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,42 @@
+class ArticlesController < AdminController
+  def new
+    category = Category.find_by(key_name: params[:key_name])
+    @translated_category = category.translation_for(current_language.id)
+
+    @conferences = category.conferences
+
+    @translated_conferences = category.conferences.map do |conference|
+      conference.translation_for(current_language.id)
+    end
+  end
+
+  def create
+    @article = Article.new(author_id: User.find_by(user_name: params[:author_user_name]).id,
+                           category_id: Category.find_by(key_name: params[:category_key]).id,
+                           conference_id: Conference.find_by(key_name: params[:conference_key]).id,
+                           team_id: params[:team_id],
+                           location_id: Location.find_by(key_name: params[:location_key])&.id,
+                           article_translations_attributes: [article_translation_params])
```

Comment:
it looks scary...

Reply:
That is why I wanted to discuss this PR simplification ways